data/json/monstergroups/mi-go.json

```diff
@@ -7,11 +7,11 @@
     "monsters": [
       { "monster": "mon_mi_go", "freq": 100, "cost_multiplier": 20, "pack_size": [ 2, 4 ] },
       { "monster": "mon_mi_go", "freq": 100, "cost_multiplier": 10, "pack_size": [ 4, 8 ] },
-      { "monster": "mon_mi_go", "freq": 100, "cost_multiplier": 2, "starts": 720, "pack_size": [ 3, 6 ] },
-      { "monster": "mon_mi_go_slaver", "freq": 30, "cost_multiplier": 10, "starts": 840 },
-      { "monster": "mon_mi_go_slaver", "freq": 30, "cost_multiplier": 20, "starts": 1080, "pack_size": [ 1, 2 ] },
-      { "monster": "mon_mi_go_scout", "freq": 100, "cost_multiplier": 50, "starts": 840 },
-      { "monster": "mon_mi_go_myrmidon", "freq": 10, "cost_multiplier": 50, "starts": 1080 }
+      { "monster": "mon_mi_go", "freq": 100, "cost_multiplier": 2, "starts": 180, "pack_size": [ 3, 6 ] },
+      { "monster": "mon_mi_go_slaver", "freq": 30, "cost_multiplier": 10, "starts": 210 },
+      { "monster": "mon_mi_go_slaver", "freq": 30, "cost_multiplier": 20, "starts": 270, "pack_size": [ 1, 2 ] },
+      { "monster": "mon_mi_go_scout", "freq": 100, "cost_multiplier": 50, "starts": 210 },
```

Comment:
I'm not sure about this (here and below). People previously were complaining about scouts spawning too yearly; they are supposed to be mid-game monsters, and this change effectively shifts them to be yearly-game ones.

Reply:
How do you define early-game and mid-game, and at what times?

Reply:
No exact times, just subjective feeling.

Early game - you have to survive: find food, water, clothes/armor, weapons and other basic stuff. You try not to engage in fights you're not absolutely sure you win. You're mostly scavenging and gathering loot, crafting is limited to stuff that will satisfy your immediate needs.
Mid-game - you survived, and can confidently survive further unless you make something stupid. You have gathered enough loot and actively crafting some serious and time consuming stuff. You found a safe place you can call your base, or a car you can call (in near future) your deathmobile, and you begin to improve them.

Reply:
The timeline of mi-go spawns was a bit all over the place. Looking into the relevant PRs what I pieced together is that Erk added them thinking `starts` takes days instead of hours (leading to scouts within the first week), abd after that confusion got cleared up wist changed them over into hours(but because of the whole evolution factor thing that means scouts in four months and myrmidons in the second summer).

In my impression a month-old survivor can already choose his engangements and has a few emergency grenades, but I can slap another month on them if you think that's more fitting.

Reply:
Month is okay.